renovate.json5

```diff
@@ -0,0 +1,94 @@
+{
+  "extends": [
+    "github>defenseunicorns/uds-common//config/renovate.json5",
+    ":semanticCommits",
+    ":semanticCommitTypeAll(chore)"
+  ],
+  "ignorePresets": [
+    ":ignoreModulesAndTests"
+  ],
+  "branchConcurrentLimit": 0,
+  "prConcurrentLimit": 0,
+  "prHourlyLimit": 0,
+  "packageRules": [
+    {
+      "matchManagers": ["npm"],
+      "matchFileNames": ["package-lock.json"],
+      "groupName": "All npm dependencies",
+      "groupSlug": "npm-dependencies"
+    },
+    {
+      "enabled": false,
+      "matchFileNames": ["src/content/docs/structure/packages.md"]
+    },
+    // Keep podinfo chart version + container image tag updates in a single PR so docs never reference mismatched versions.
+    {
+      "groupName": "podinfo docs",
+      "groupSlug": "podinfo-docs",
+      "matchManagers": ["custom.regex"],
+      "matchDepNames": [
+        "https://github.com/stefanprodan/podinfo.git",
+        "ghcr.io/stefanprodan/podinfo"
+      ]
+    }
+  ],
+  "customManagers": [
+    // Matches a ghcr.io repository (including oci://) and captures the next ref version, even if it spans multiple package blocks.
+    {
+      "customType": "regex",
+      "managerFilePatterns": [
+        "/^src\\/content\\/docs\\/.*\\.(md|mdx)$/"
+      ],
+      // Test: https://regex101.com/r/2bifNc/
+      "matchStrings": [
+        "repository:\\s*(?:(?:oci://)?(?<registry>ghcr\\.io)/(?<depPath>[^\\s]+))[\\s\\S]*?ref:\\s*(?<currentValue>[^\\s\"']+)"
+      ],
+      "datasourceTemplate": "docker",
+      "depNameTemplate": "{{registry}}/{{depPath}}",
+      "versioningTemplate": "regex:^v?(?<major>\\d+)(\\.(?<minor>\\d+))?(\\.(?<patch>\\d+))?",
+      "extractVersionTemplate": "^(?<version>v?\d+\.\d+(?:\.\d+)?)(?<suffix>-(?:uds\.\d+(?:-[\w.-]+)?|[\w.-]+))?$",
+      "autoReplaceStringTemplate": "{{{newValue}}}{{{suffix}}}",
+      "registryUrlTemplate": "https://{{registry}}"
+    },
+    // Matches a ghcr.io repository and captures its ref version, limited to the same package entry only.
+    {
+      "customType": "regex",
+      "managerFilePatterns": [
+        "/^src\\/content\\/docs\\/.*\\.(md|mdx)$/"
+      ],
+      // Test: https://regex101.com/r/Zpqw2u/
+      "matchStrings": [
+        "(?m)^\\s*-\\s+['\"]?(?<depName>(?:ghcr\\.io|quay\\.io|docker\\.io)/[^\\s:@'\"]+(?:/[^\\s:@'\"]+)*)['\"]?:(?<currentValue>[\\w.\\-]+)\\s*$"
+      ],
+      "datasourceTemplate": "docker",
+      "versioningTemplate": "docker"
+    },
+    // Matches GitHub .git dependencies in markdown YAML blocks by pairing a `version:` field with its repo URL
+    {
+      "customType": "regex",
+      "managerFilePatterns": [
+        "/^src\\/content\\/docs\\/.*\\.(md|mdx)$/"
+      ],
+      // Test: https://regex101.com/r/mBEs5T/
+      "matchStrings": [
+        "(?m)name:\\s*[^\\n]+[\\s\\S]*?version:\\s*(?<currentValue>\\d+\\.\\d+\\.\\d+)[\\s\\S]*?url:\\s*[\"']?(?<gitUrl>https?://(?:github\\.com)/[^\\s\"']+?\\.git)[\"']?\\s*$"
+      ],
```

Comment:
Looking at the preview (regex101) this seems to match the wrong version? It catches the one in metadata rather than the chart one.